src/transactions/feed/NftFeedItem.tsx

```diff
@@ -39,15 +90,22 @@ function NftFeedItem({ transaction }: Props) {
   return (
     <Touchable testID={'NftFeedItem'} disabled={false} onPress={openNftTransactionDetails}>
       <View style={styles.container}>
-        {transaction.type === TokenTransactionTypeV2.NftReceived ? (
+        {/* If enabled try to show the first image. Otherwise display the default icons */}
+        {showNftsInApp && nfts.length > 0 && nfts[0].metadata?.image ? (
+          <NftIcon nft={nfts[0]} />
+        ) : transaction.type === TokenTransactionTypeV2.NftReceived ? (
           <NftReceivedIcon />
         ) : (
           <NftSentIcon />
         )}
         <View style={styles.descriptionContainer}>
           <Text style={styles.title}>
             {transaction.type === TokenTransactionTypeV2.NftReceived
-              ? t('receivedNft')
+              ? nfts.length > 1
+                ? t('receivedNfts')
+                : t('receivedNft')
+              : nfts.length > 1
+              ? t('sentNfts')
               : t('sentNft')}
```

Comment:
i18next supports [plurals](https://www.i18next.com/translation-function/plurals), we should use it instead.

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/3889/commits/ff6263382977d369abebfeaaac86f1fd721f747a!